docs/elasticsearch.md

```diff
@@ -48,3 +48,17 @@ Copy the mapped IP and port (highlighted in bold above) and use it to query Elas
 ```
 curl -XGET http://0.0.0.0:32871
 ```
+
+## Configuration updates when using ElasticSearch on Local Environment
+
+ElasticSearch uses more memory within the local server environment, which can also cause services to stop working,n particularly when using Ubuntu. Altis includes an environment variable which can change the default memory limit for ElasticSearch called `ES_MEM_LIMIT`. 
+
+### Change variable during current session only
+
+Using this variable with the `composer server start` command, users can update the memory limit for the current session only; it won't change the variable permenantly. An example might be: 
```

Comment:
We're addressing the individual here, they're reading the docs to find out how to do something so it's a bit odd to read "users can update". This is something the reader needs to do.

Reply:
So maybe instead
```Update the memory limit for the current session only by using this variable with the `composer server start` command; it won't change the variable permanently. For example: ```